build.sbt

```diff
@@ -5,7 +5,10 @@ ThisBuild / tlBaseVersion   := "0.1"
 ThisBuild / tlCiHeaderCheck := false
 
 val commonSettings = Seq(
-  libraryDependencies += "org.creativescala" %% "krop-core" % "0.4.1"
+  libraryDependencies ++=
+    "org.creativescala" %% "krop-core" % "0.6.0" ::
+      Modules.circe :::
+      Nil
```

Comment:
Why is this list appended to `Nil`? It does nothing. I found this code very confusing on first read.

Reply:
You are right ... don't need `Nil` as `Modules.circe` is a `List` already.